### ISSUE TITLE
sign up popup appears way below the mobile phone screen 

### ISSUE THREAD

**Issue by susanreenesaa** [publiclab/plots2#6796]
### Description of the problem (or idea)
The signup popup appears way below the mobile phone screen. This makes the user think that after clicking signup, only a dark screen opened. However when you scroll down, one is able to see the signup page as shown in the pictures below.
**Before Scrolling down**
![image](https://user-images.githubusercontent.com/49034794/69058239-59ce1580-0a24-11ea-9022-27f68f6378bc.png)
**After scrolling**
![image](https://user-images.githubusercontent.com/49034794/69058288-710d0300-0a24-11ea-8294-c7cdd96f77cf.png)

**Expected**
The user to be able to see the popup screen before scrolling down the page

### Where to look
any URL will work. In the menu click on signup i.e
https://publiclab.org/

### What's your PublicLab.org username?
Reenesa
> This can help us diagnose the issue: 

### Browser, version, and operating system
Google Chrome
Ubuntu 18.04.3 LTS

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!
****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by Asiedu13** [comment]
Can I claim the task?

**Comment by NitinBhasneria** [comment]
Duplicate of #6572. You should check this issue it is way similar and so You should close this issue.

**Comment by NitinBhasneria** [comment]
@jywarren Kindly have a look at this issue. As after #7261 this also needs to be closed.

**Comment by jywarren** [comment]
Ah, thank you for the catch! Closing! Thanks, all!
